frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/InteractiveEmbeddingCTA/InteractiveEmbeddingCTA.tsx

```diff
@@ -36,7 +36,15 @@ const useCTAText = () => {
     // eslint-disable-next-line no-literal-metabase-strings -- This only shows for admins
     description: t`Give your customers the full power of Metabase in your own app, with SSO, advanced permissions, customization, and more.`,
     linkText: t`Learn more`,
-    url: `https://www.metabase.com/product/embedded-analytics?utm_source=${plan}&utm_media=static-embed-popover`,
+    url: `https://www.metabase.com/product/embedded-analytics?${new URLSearchParams(
```

Comment:
Every place is using `URLSearchParams` with an object initializer to make UTM code more readable than a long string.

Reply:
(TIL that `URLSearchParams` implicitly casts to string params by default)